src/sentry/event_manager.py

```diff
@@ -1169,7 +1167,7 @@ def _save_aggregate(event, hashes, release, metadata, received_timestamp, **kwar
         # _save_aggregate had races around group creation which made this race
         # more user visible. For more context, see 84c6f75a and d0e22787, as
         # well as GH-5085.
-        GroupHash.objects.filter(id__in=[h.id for h in new_hashes]).exclude(
+        GroupHash.objects.filter(id__in=[h.id for h in new_grouphashes]).exclude(
```

Comment:
@untitaker This now unconditionally adds the group ID to every newly created `GroupHash`, but I'm not sure if that was the original intention.